doc/search-metrics.org

```diff
@@ -59,6 +59,17 @@ For instance you can search malicious judgments with high confidence matching "a
 GET /ctia/incident/search?query=abuse.ch&confidence=high&limit=10&sort_by=disposition&sort_order=desc
 #+END_SRC
 
+Finally you can only query entity count with the same filtering parameters through the endpoint ~GET /ctia/{entity-type}/search/count~.
+So reusing a previous search query example, one can count the indicators with "android" in the title, having a high confidence, created since 2020/01/01 with both text and term filtering:
+#+BEGIN_SRC
+GET /ctia/indicator/search/count?query=title:android&confidence=high&from=2020-01-01
```

Comment:
is `:` an ES notation? Because the most usual notation is `.`.

Reply:
it's a lucene notation inside the `query` text field, it will match entities with `title` field that matches `android`.